core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4853,7 +4853,7 @@ public String toString() {
       new Builder(Name.USER_STREAMING_DATA_READ_TIMEOUT)
           .setAlias("alluxio.user.network.data.timeout.ms", Name.USER_NETWORK_DATA_TIMEOUT,
               Name.USER_STREAMING_DATA_TIMEOUT)
-          .setDefaultValue("1h")
+          .setDefaultValue("5m")
```

Comment:
I guess the "problem" behind this property is, it doesn't really timeout easily in our existing tests. So when there are problems, they most likely take place in a very large production workload. So maybe it's hard to just update this value and try our tests. A potentially better way is to rely on people's experience (yea not too reliable, i know).

Reply:
I don't think there's any compute app out there that would tolerate 1h wait time. They usually timeout requests between 5-8 mins. So our defaults should be reflective of that.

After considering more, I set this to 3m and I believe that should be the most as the default value. If there are clusters out there that have very slow connections, the way should be tuning them individually not via bloated defaults like we have here.